draft-demarco-oauth-status-attestations.md

```diff
@@ -133,27 +133,23 @@ validity of the Digital Credentials, including their revocation status,
 before accepting them.
 
 Wallet Instance:
-: The digital Wallet in control of a User, also known as Wallet or Holder.
+: The digital Wallet in control of a User, also known as Wallet.
 It securely stores the User's Digital Credentials. It can present
 Digital Credentials to Verifiers
 and request Status Assertions from Issuers under the control of the User.
 
 # Rationale
 
-OAuth Status Lists [@!I-D.looker-oauth-jwt-cwt-status-list] are suitable
-for specific scenarios, especially when the Verifier needs to verify the
-status of a Digital Credential at a later time after the User has presented the
-Digital Credential.
-
```

Comment:
I think its okay to keep this one in

Reply:
For the scopes of this PR, the rationale about the status assertion takes out any comparison to Status List.